cfgov/teachers_digital_platform/jinja2/teachers_digital_platform/survey/results-3-5.html

```diff
@@ -27,6 +27,13 @@
         <p class="h1">Congratulations, {{ overall.heading_html|safe }}</p>
         <p class="h3"> {{ overall.msg_html|safe }}</p>
         {% set car_position = ((overall.position_idx +1) * 140) - 50 %}
+        {% if car_position < 2 >= 0 %}
+            {% set car_position = (car_position * 100) + (40) %}
+        {% elif car_position < 5 >= 2 %}
+            {% set car_position = (car_position * 100) + (100) %}
+        {% else %}
+            {% set car_position = (car_position * 100) + (150)  %}
+        {% endif %}
```

Comment:
May I suggest using a unique var name for these pixel values, and moving its calculation inside the template?